core/src/main/R/base/strwrap.R

```diff
@@ -191,3 +191,16 @@ function(x, y, style = c("table", "list"),
     }
     r
 }
+
+trimws <- function(x, which = c("both", "left", "right"),
+                   whitespace = "[ \t\r\n]")
```

Comment:
Is there a `whitespace` argument defined in GNU R? I'd prefer to keep the base function API exactly the same.

Reply:
Yes. The function is just mirrored from the latest R version (`3.5.2`).